docker/Dockerfile

```diff
@@ -46,7 +54,8 @@ RUN cd /opt/lmdeploy &&\
     ninja -j$(nproc) && ninja install &&\
     cd .. &&\
     python3 -m pip install -e . &&\
-    rm -rf build
+    rm -rf build  &&\
+    rm -rf ~/.cache
```

Comment:
why remove "~/cache" again

Reply:
> why remove "~/cache" again

for remove the pip cache and reduce image size;

Reply:
Can't we keep only one "rm -rf ~/.cache"?

Reply:
> Can't we keep only one "rm -rf ~/.cache"?

bacause union file system in docker, every image level(every command RUN or files change in Dockerfile) is independent, Changes to the next level will not materially affect the files of the previous layer;  if i only execute "rm -rf ~/.cache" at the fifth RUN will not delete the cache files of the second RUN;